src/components/cards/BaseCardPool.vue

```diff
@@ -19,7 +19,7 @@
                 </p>
                 <base-modal-delete :id="`modalDelete-${pool.address}`" :call="() => remove()" :subject="pool.address" />
                 <hr />
-                <b-button :disabled="outOfDate" class="rounded-pill" variant="primary" @click="openPoolUrl()" block>
```

Comment:
@GarfDev Why did you remove this attribute? It is important that users don't get access to pools where the contract implementation is running behind on the latest version of our contract layer.